ngraph/core/include/ngraph/op/util/sub_graph_base.hpp

```diff
@@ -349,6 +349,10 @@ namespace ngraph
     }
     template class NGRAPH_API FactoryRegistry<op::util::SubGraphOp::InputDescription>;
 
+    template <>
+    FactoryRegistry<op::util::SubGraphOp::InputDescription>&
```

Comment:
Why do we need this specialization?

Reply:
CC @emmanuelattia-philips

Reply:
We need to specialize explicitely to overcome an unresolved external when compiling ngraph statically, it has no impact when compiling ngraph dynamically.